osmosis-1/osmosis-1.assetlist.json

```diff
@@ -1497,6 +1497,34 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/polkadot/images/dot.svg"
       },
       "coingecko_id": "polkadot"
+    },
+    {
```

Comment:
please remove listing from this file. It should only be in the frontier file for now.

Reply:
@JeremyParish69 Done. For clarification purposes, how can we list the token on the main frontend?

Reply:
@diogofferreira To list on Osmosis Frontier, a PR is required to this frontend repo. E.g., https://github.com/osmosis-labs/osmosis-frontend/pull/723. To list on the Main Osmosis app, there should first be a Cudos pool with Osmosis bonding incentives, which must be approved by governance.